math/src/elliptic_curve/short_weierstrass/curves/bls12_381/compression.rs

```diff
@@ -106,7 +102,48 @@ impl Compress for BLS12381Curve {
             .ok_or(ByteConversionError::PointNotInSubgroup)
     }
 
-    #[allow(unused)]
+    #[cfg(feature = "alloc")]
+    fn compress_g2_point(point: &Self::G2Point) -> alloc::vec::Vec<u8> {
```

Comment:
Do we really need a `Vec` here or is it constant size?

Reply:
Traits get in the way, but we can try to solve this if we expose more things